### ISSUE TITLE
GM Jump/Call issues

### ISSUE THREAD

**Issue by JeidoUran** [SapphireServer/Sapphire#341]
GM Jumping to a character within an instance while being outside of said instance crashes the zone server. The same occurs if a character within an instance GM Calls a character that is outside of said instance. The issue occurs even if both characters are properly bound to the instance.

Also, a very minor issue I noticed, but since you guys want to be as close as possible to retail behavior, I figured I'd mention it:

On Sapphire, when GM Jumping/Calling a character that is in the same zone as the player using the command, it will not trigger a loading screen. However, on retail, I've been teleported by GMs several times, and even within the same zone, it always triggered a loading screen.

**Comment by NotAdam** [comment]
i guess it's more lore friendly to have a load screen than instantaneously move across the zone - just a matter of sending another initzone packet iirc to trigger a load screen

**Comment by JeidoUran** [comment]
//gm call is now kind of broken. Here's the test I did with 2 players in the same zone:

1) Player 1 calls Player 2
2) Player 2 gets a loading screen, gets teleported to Player 1
3) Either nothing happens and it works, or Player 2 stops seeing Player 1
4) I tried to use //gm invis with Player 1 to force a respawn
5) It completely despawned Player 2, who couldn't do anything.

Also, teleporting a player outside of an instance was already semi-working before (the called player would be properly teleported but would still have the music, duty list, timer, afk timer, etc of their instance). Calling a player inside one (which is now fixed/disabled) and jumping to a player inside one (still broken) is what was broken. Sorry if I was unclear.
